.travis.yml

```diff
@@ -6,8 +6,21 @@ cache:
   directories:
    - $HOME/.m2
 
-# skip install
-install: true
+install:
+  # download Cloud SDK
+  - wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-112.0.0-linux-x86_64.tar.gz
+  - tar -xzvf google-cloud-sdk-112.0.0-linux-x86_64.tar.gz
+  - export PATH=$PWD/google-cloud-sdk/bin:$PATH
```

Comment:
can we also export $CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true so we don't mess with metrics reporting?